VideoLocker/src/test/java/org/edx/mobile/view/UserProfileFragmentTest.java

```diff
@@ -0,0 +1,76 @@
+package org.edx.mobile.view;
+
+import android.databinding.DataBindingUtil;
+import android.view.View;
+
+import org.edx.mobile.R;
+import org.edx.mobile.databinding.FragmentUserProfileBinding;
+import org.edx.mobile.profiles.UserProfilePresenter;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.CoreMatchers.nullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.robolectric.Shadows.shadowOf;
+
+public class UserProfileFragmentTest extends PresenterFragmentTest<UserProfileFragment, UserProfilePresenter, UserProfilePresenter.ViewInterface> {
+
+    FragmentUserProfileBinding binding;
+
+    @Before
+    public void before() {
+        startFragment(UserProfileFragment.newInstance("not used"));
+        binding = DataBindingUtil.getBinding(fragment.getView());
+        assertThat(binding, is(notNullValue()));
+    }
+
+    @Test
+    public void setName_updatesTextView() {
+        final String text = "hello";
+        view.setName(text);
+        assertThat(binding.nameText.getText().toString(), is(text));
```

Comment:
Note that we're using the AssertJ Android library, which has specific assertion utility methods defined for all common Java and Android framework methods.